src/index.js

```diff
@@ -117,13 +129,13 @@ exports.decode = function decode (buf) {
  * @param {number} [length]
  * @returns {Buffer}
  */
-exports.encode = function encode (digest, code, length) {
+export function encode (digest:Buffer, code:Code, length:number):Multihash {
```

Comment:
I'm working on converting `js-multihashing` and I think that code parameter here should be `code: Code | Name`
```
export function encode (digest:Buffer, code:Code | Number, length:number):Multihash {
```